package.json

```diff
@@ -27,18 +28,23 @@
     "**/webpack**",
     "**/babel-loader**"
   ],
+  "private": false,
   "scripts": {
     "predeploy": "yarn run build",
     "start": "concurrently \"cd matico_components && yarn start\" \"cd editor && yarn start\" ",
     "dev-editor": "concurrently \"cd matico_components && yarn build-dev\" \"cd editor && yarn start\" ",
     "build": "yarn workspace @maticoapp/matico_charts build-prod && yarn workspace @maticoapp/matico_components build-prod &&  (cd matico_app_server && yarn run prisma generate && yarn build )",
     "deploy": "yarn predeploy && yarn run netlify deploy --prod",
     "build_rust_docs": "cargo doc --open --document-private-items --no-deps",
+    "postinstall": "husky install",
+    "prepublishOnly": "pinst --disable",
```

Comment:
Is this just to prevent husky from loading in certain situation?

Reply:
> if your package is not private and you're publishing it on a registry like [npmjs.com](https://npmjs.com/), you need to disable postinstall script using [pinst](https://github.com/typicode/pinst). Otherwise, postinstall will run when someone installs your package and result in an error.

This appears to be the reason. For the next step in this process I'll put the hook in and then do a build. If you'd like me to do any other stop, let me know and I'll have that as part of the testing.